src/render_cmd.rs

```diff
@@ -321,6 +370,7 @@ mod tests {
                 helm_bin: None,
                 additional_options: None,
                 update_dependencies: false,
+                filter: vec![],
```

Comment:
```suggestion
filter: None
```

Since you defined `filter` to be an option for a string.